internal/driver/driver.go

```diff
@@ -96,7 +114,7 @@ func (d *Driver) createClientOptions() ([]opcua.Option, error) {
 	policy := ua.SecurityPolicyURIBasic256Sha256
 	mode := ua.MessageSecurityModeSignAndEncrypt
 
-	ep := opcua.SelectEndpoint(availableServerEndpoints, policy, mode)
+	ep := SelectEndPoint(availableServerEndpoints, policy, mode)
```

Comment:
same as above

Reply:
This was done for mocking, by doing this I can substitute GetEndpoints with a mock in unit tests if I want to test createClientoptions. I added a comment to make it clearer